src/main/java/jenkins/plugins/purgejobhistory/PurgeJobHistory.java

```diff
@@ -0,0 +1,49 @@
+package jenkins.plugins.purgejobhistory;
+
+import hudson.Extension;
+import hudson.cli.CLICommand;
+import hudson.model.Item;
+import hudson.model.Job;
+import hudson.model.Run;
+import hudson.model.TopLevelItem;
+import java.util.ArrayList;
+import org.kohsuke.args4j.Argument;
+import org.kohsuke.args4j.Option;
+
+/**
+ * @author Stephen Connolly
+ */
+@Extension
+public class PurgeJobHistory extends CLICommand {
+    
+    /**
+     * Follows the progress of the operation
+     */
+    @Option(name = "-r", usage = "Also reset the next build number.")
+    public boolean reset = false;
+
+    /**
+     * The source item.
+     */
+    @Argument(metaVar = "JOB", usage = "Name of the job to purge", required = true)
```

Comment:
Additionally "whose *build* history", as opposed to [job *config* history](https://wiki.jenkins-ci.org/display/JENKINS/JobConfigHistory+Plugin) or any other type of history.